src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -716,7 +717,7 @@ public boolean equals(Object o) {
                 Objects.equals(jenkinsUrl, that.jenkinsUrl) &&
                 Objects.equals(jenkinsTunnel, that.jenkinsTunnel) &&
                 Objects.equals(credentialsId, that.credentialsId) &&
-                Objects.equals(podLabels, that.podLabels) &&
+                Objects.equals(getPodLabels(), that.getPodLabels()) &&
```

Comment:
This is what I should have done in the first attempt.

Reply:
Ah, clear now.